src/main/kotlin/me/alexflipnote/kawaiibot/commands/Lick.kt

```diff
@@ -20,8 +21,7 @@ class Lick : ICommand {
             m.user.idLong == ctx.jda.selfUser.idLong ->
                 ctx.send("${ctx.author.name}... w-why do you lick me ;-;")
             m.user.idLong == ctx.author.idLong ->
-                // TODO: download and bundle this image
-                ctx.send("https://blog.eat24.com/wp-content/uploads/2016/03/cat-licking-itself-.gif")
+                ctx.channel.sendFile(Helpers.getImageStream("images/selflick.gif"), "selflick.gif").queue()
```

Comment:
Why not use an embed the original URL? It would be less resource intensive than uploading the image every time.

Reply:
It's better to be consistent with the other commands which also upload their own images every time in my opinion, changing all those would be a completely different PR.

Reply:
👌

Reply:
^Plus links look somewhat unclean. Imagine responses like `Oh you're all alone? Sorry to hear <link to image of gif>`

Reply:
I think you misunderstood what I meant by "embed". I don't mean just plopping the URL down, I mean using the actual rich embed structure that discord provides.